pom.xml

```diff
@@ -30,11 +30,11 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <version.jdk>1.8</version.jdk>
-        <version.log4j>2.0.2</version.log4j>
-        <version.jmh>1.9.1</version.jmh>
+        <version.log4j>2.3</version.log4j>
+        <version.jmh>1.9.3</version.jmh>
 
-        <version.thrift>0.9.1</version.thrift>
-        <version.protobuf>2.5.0</version.protobuf>
+        <version.thrift>0.9.2</version.thrift>
+        <version.protobuf>2.6.1</version.protobuf>
```

Comment:
Can you fix the README to match these versions? (thrift and protoc executables must match libraries).